packages/flutter_tools/test/integration.shard/test_data/hot_reload_project.dart

```diff
@@ -25,12 +25,21 @@ class HotReloadProject extends Project {
   import 'package:flutter/scheduler.dart';
   import 'package:flutter/services.dart';
   import 'package:flutter/widgets.dart';
+  import 'package:flutter/foundation.dart';
 
   void main() async {
     WidgetsFlutterBinding.ensureInitialized();
     final ByteData message = const StringCodec().encodeMessage('AppLifecycleState.resumed')!;
     await ServicesBinding.instance!.defaultBinaryMessenger.handlePlatformMessage('flutter/lifecycle', message, (_) { });
-    runApp(MyApp());
+    // See https://github.com/flutter/flutter/issues/86202
```

Comment:
To make sure I understand, the solution to prints getting dropped that is being used here is to just call `runApp` repeatedly? When does it stop?

Reply:
yeah, run app causes a rebuild which triggers more prints. It stops when the app is stopped